### ISSUE TITLE
Incorrect link color inside dark theme accordion fails WCAG 2.1, SC 1.4.3 requirements

### ISSUE THREAD

**Issue by aorlova2909** [designsystemau/gold-design-system-site#14]
# Bug report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/designsystemau/design-system-site/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/designsystemau/design-system-site/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

In 'Overview' tab, in dark mode example, the link 'is' inside the accordion is blue #0000EE on blue background #135E70 which gives color contrast ratio of 1.28:1.

## What I expected to happen

The contrast ratio of the text against the background should be at least 4.50:1 for small text and 3.00:1 for large text (14pt bold or 18 pt+).
See correct implementation in 'Accessibility' tab, 'Keyboard accessibility' example.

## Reproducing

- Website URL: https://gold.designsystemau.org/components/accordion/
- Browser: Chrome
- Operating system: Windows

Steps to reproduce:

1. Open https://gold.designsystemau.org/components/accordion/
2. In 'Overview' tab expand the dark theme accordions
3. Measure the color contrast ratio of blue 'is' link on blue background

## Attachments

![goldDS-accordion-blue link on blue](https://user-images.githubusercontent.com/89005205/139830330-237d13db-91bc-47bc-b824-f2b8ea8ff0f6.png)

**Comment by dominikwilkowski** [comment]
This is actually like this on purpose as text content inside the accordion should be wrapped in [body](https://gold.designsystemau.org/components/body/) which would fix this issue. But I do see this as not a great experience in the docs so maybe we should have the docs use the body component as well to make that clear? It might just be confusion about since it is not needed.
